integrated_channels/blackboard/migrations/0002_auto_20200930_1723.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.16 on 2020-09-30 17:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('blackboard', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='blackboardenterprisecustomerconfiguration',
+            name='client_id',
```

Comment:
where are we storing the token? Just in the session? Ie how would it be accessible to Django Admin if it's not in the db?

Reply:
there is a `refresh_token` field in the db https://github.com/edx/edx-enterprise/pull/980/files#diff-444de76cae0d55485100349a6900b8e5R52 but we only store the refresh_token in the db

the access token is stored in the sessoin under 

`session.headers['Authorization'] = 'Bearer {}'.format(oauth_access_token)`

and used until it expires after which we refetch using that last fetched refresh_token stored in the db

Reply:
Hmm that diff doesn't seem to really work or at least in the files for this pr I do not see refresh token on the model nor the migration. Was it already defined in your previous pr? If so, you can ignore me here. :)

Reply:
Just checked, looks like the refresh token field's already in master.

Reply:
Thank you for doing what I was too lazy to do

Reply:
yeah sorry should have explained that